drivers/hmis/spec/models/hmis/form/form_processor_spec.rb

```diff
@@ -2004,6 +2004,19 @@ def process_assessment(new_hud_values: nil)
       expect(cls.information_date).to eq(information_date)
     end
 
+    it 'should work when verified by project ID is provided' do
+      hud_values = {
+        'CurrentLivingSituation.informationDate' => information_date.strftime('%Y-%m-%d'),
+        'CurrentLivingSituation.currentLivingSituation' => 'SAFE_HAVEN', # 118
+        'CurrentLivingSituation.verifiedByProjectId' => p1.id,
+      }
+      cls = Hmis::Hud::CurrentLivingSituation.new(client: c1, enrollment: e1, data_source: ds1)
+      process_record(record: cls, hud_values: hud_values, user: hmis_user, definition: definition)
+
+      expect(cls.verified_by_project_id).to eq(p1.id)
+      expect(cls.verified_by).to eq(p1.name)
```

Comment:
recently I've been thinking we should make more use of expect blocks to checks that the actions are actually changing the value. Perhaps:
```suggestion
      expect do
        process_record(record: cls, hud_values: hud_values, user: hmis_user, definition: definition)
      end.to change(cls, :verified_by_project_id).to(p1.id).
      and change(cls, :verified_by).to(p1.name)
```

Reply:
Thanks, agree this is better!